### PR TITLE
Feature Base Video Controls Prop

### PR DESCRIPTION
**Pull request by mdmullendore**
### SUMMARY

Adds controls prop to `base-video` component.

### SCREENSHOT
Screen Shot 2022-01-05 at 3 10 26 PM

### CODE REVIEW

File: .node-version [hunk 1]
```diff
@@ -0,0 +1 @@
+10.16.0
```
Comment (mdmullendore) [inline .node-version:1]: This seems to be the desired node version for sass.

File: packages/vue/index/components/base/ZrVideo.vue [hunk 2]
```diff
@@ -89,6 +91,13 @@
       videoType: {
         type: String,
         default: 'video/mp4'
+      },
+      /**
+       * Whether or not to show the video controls
+       */
+      controls: {
+        type: Boolean,
+        default: false
```
Comment (tylerrzaneray) [inline packages/vue/index/components/base/ZrVideo.vue:100]: Just a note that whenever `controls` is set to false, the video will not be accessible. Is it worth setting controls to true by default to improve accessibility?

Comment (mdmullendore) [reply packages/vue/index/components/base/ZrVideo.vue:100]: The component now is given the option to make the video accessible. With out the controls attr the video controls do not appear.

Comment (tylerrzaneray) [reply packages/vue/index/components/base/ZrVideo.vue:100]: I think it's great that the controls attribute has been added, but my question is: why would we set the default to "not accessible". In theory, there will never be an instance where the "false" option is used, so it seems odd that we'd make that the default. It's akin to making the default image have no alt attribute and requiring that devs remember to manually add it every time.

Comment (mdmullendore) [reply packages/vue/index/components/base/ZrVideo.vue:100]: For overhead purposes. We would then have to go through each repo that uses the video component and adjust the controls accordingly.

**Pull request merged**